runtime/build.gradle.kts

```diff
@@ -50,14 +60,32 @@ kotlin {
             }
         }
 
+        val androidMain by getting {
+            kotlin.srcDir("src/commonJvmAndroid/kotlin")
+            dependencies {
+                api("com.google.protobuf:protobuf-java:${Versions.protobufJava}")
+            }
+        }
+
+        val androidTest by getting {
+            dependencies {
+                implementation(kotlin("test"))
+                implementation(kotlin("test-junit"))
+                implementation("junit:junit:4.12")
+                runtimeOnly("org.robolectric:android-all:10-robolectric-5803371")
```

Comment:
Please add a comment that we shouldn't bump this version higher than `10` until Android Studio is using Java 11 instead of Java 8.

Reply:
I'll also move this version to `Versions.kt`